SpooderShooter/Assets/Scripts/Players/Movement.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+
+public class Movement : MonoBehaviour {
+    private float speed = 1;
+    private CharacterController controller; 
+
+	// Use this for initialization
+	void Start () {
+        controller = GetComponent<CharacterController>();
+    }
+
+    // Update is called once per frame
+    void Update () {
+        CharacterController controller = GetComponent<CharacterController>();

```

Comment:
You can remove this line because you already defined this as a member.

Reply:
Oh man, how'd I miss that XD.

This is why we code review!!